style.css

```diff
@@ -0,0 +1,67 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700;800&family=Roboto&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  height: 718px;
+}
+
+.header-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  padding: 58px 32px 0 24px;
+}
+
+.logo {
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  text-decoration: none;
+}
+
+.first-container {
+  display: flex;
+  background-image: url(./images/mobile-images/mobile-background.png);
+  background-repeat: no-repeat;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 24px;
+  gap: 12px;
+  border-bottom-left-radius: 90px;
+}
+
+.intro-text {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.intro-msg {
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  color: #344563;
+  text-align: left;
+}
+
+.connect {
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.social {
+  display: flex;
+  justify-content: flex-start;
+  gap: 12px;
+}
```

Comment:
- [ ] While implementing the changes suggested by the previous reviewer, a new bug 🐞 was introduced in your code, causing a mismatch between your UI design and the design provided in `Figma`. Upon investigation, it appears that the addition of the `li` element in your HTML is the main cause of this issue. The `li` element comes with its own default styles, which conflict with the desired design. To resolve this issue, you should consider overriding the default styles applied to the `li` element by adding a `list-style: none` and adjusting the spacing between your icons to match that of the `Figma` design. See below for more details:
<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="217" alt="Screenshot 2023-07-05 at 16 09 23" src="https://github.com/Kwamay/My-Porforlio/assets/107192076/ed3f3427-674d-4c8e-93f4-42885e53a290">
</td>
<td>
<img width="213" alt="Screenshot 2023-07-05 at 16 08 01" src="https://github.com/Kwamay/My-Porforlio/assets/107192076/c89d70c5-9077-49f1-9559-1ac2a2ccd8f2">
</td>
</tr>
</table>